src/components/wind-circular-control.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import { useStores } from "../use-stores";
+import {
+  CircularInput,
+  CircularTrack,
+} from "react-circular-input";
+import WindDial from "../assets/wind-dial.svg";
+import WindArrow from "../assets/wind-arrow.svg";
+import WindSymbol from "../assets/wind-symbol.svg";
+import css from "./wind-circular-control.scss";
+import { Slider } from "@material-ui/core";
+import HorizontalHandle from "../assets/slider-horizontal.svg";
+
+const windSpeedMarks = [
+  {
+    value: 0,
+    label: "0"
+  },
+  {
+    value: 10,
+    label: "10"
+  },
+  {
+    value: 20,
+    label: "20"
+  },
+  {
+    value: 30,
+    label: "30"
+  }
+];
+export const WindCircularControl = () => {
+  const { simulation, ui } = useStores();
+  const [directionAngle, setDirectionAngle] = useState(simulation.wind.direction / 360);
+  const [windSpeed, setWindSpeed] = useState(simulation.wind.speed);
+
+  useEffect(() => {
+    simulation.setWindDirection(angleToDirection());
+    simulation.setWindSpeed(windSpeed);
+  }, [directionAngle, windSpeed]);
```

Comment:
Hooks, nice. 😉 
In this case, `useState` and `useEffect` might be redundant. Note that we use exactly the same variables that are already in the store (transformation of the value don't change much here). And `useStores` hook should already ensure that component gets re-rendered (this func re-executed) when some of its properties get updated.